Tests/LibWeb/TestConfig.ini

```diff
@@ -4,6 +4,7 @@ Text/input/Crypto/SubtleCrypto-exportKey.html
 Text/input/Crypto/SubtleCrypto-generateKey.html
 Text/input/WebAnimations/misc/DocumentTimeline.html
 Text/input/Worker/Worker-blob.html
+Text/input/Worker/Worker-close-after-postMessage.html
```

Comment:
I've preemptively added this to the list of skipped tests, because all worker tests are currently flaky on MacOS CI.

Reply:
Makes me wonder if it makes sense to have a platform skip option at some stage given MacOS always seems like the flakey one